### DISCUSSION TITLE
Images not downloading to Obsidian

### DISCUSSION THREAD

**Discussion by Grant837** [deathau/markdownload#234] [General]
Hi, I am using Obsidian (latest version) on a Windows 11 PC using Firefox, and, of course the Markdown extension.  I really like how it works, but there is no way I can figure out how to get the images to download.  Right now, if I use anything other than linking to the original source option, I get a message on the page in Obsidian that similar to "https://www.popularwoodworking.com/wp-content/uploads/2020/01/aw_13352_015-2-e1637095789109.jpg" is not created yet. Click to create."   I am using the Send tab to Obisidian´ option in the popup menu 

Can someone help me to get this to work?

I am using the URI plugin
I have download images on, 
I have enabled Obisidian integration
Download APIs is On
I have the I have the image file name prefix set to "{pageTitle}/"  although I have no idea how this works in practice

I have tried most if not all combinations of Image Style and Image Reference style with no success

Thanks!

**Comment by webtail** [comment]
Same issue here. The images are getting downloaded however but are not showing up in the imported .md
![1](https://github.com/deathau/markdownload/assets/43131449/843f3e3c-e561-4b8e-8f7a-fab782bb819f)

Help apppreciated :) 
Thxs

**Comment by Grant837** [comment]
Alas, I fear this plugin, necessary to make Obsidian a good note-taking/information manager / second brain,  has been abandoned, as I have now abandoned Obsidian - too many erratic quirks with the necessary plugins for more than just writing notes.
